common/src/main/java/org/ergoplatform/appkit/AppkitProvingInterpreter.scala

```diff
@@ -244,15 +243,16 @@ class AppkitProvingInterpreter(
       val proverResult = proveReduced(reducedInput, unsignedTx.messageToSign)
       val signedInput = Input(unsignedInput.boxId, proverResult)
 
-      currentCost = addCostLimited(currentCost, proverResult.cost, maxCost, msg = signedInput.toString())
+      // TODO add verification cost here
```

Comment:
not sure if it is good to add TODOs with this PR. maybe it should better wait until sigmastate has added the needed parts?

Reply:
Implemented instead. This was a missing part of cost estimation.